src/course-unit/xblock-container-iframe/index.scss

```
@@ -0,0 +1,4 @@
+.xblock-container-iframe {
+    width: calc(100% + ($spacer * .3125));
+    margin: 0 (-($spacer * .3125));
```

[optional]: Since this MFE will be moving to the use of Paragon Design tokens, it would be great to already envisage the use of CSS variables in such mathematical constructions.

```suggestion
    margin: 0 (($spacer * .3125) * -1);
```

---

Resolved